src/components/interactive-map.js

```diff
@@ -23,16 +23,6 @@ const propTypes = Object.assign({}, StaticMap.propTypes, {
   // Min pitch in degrees
   minPitch: PropTypes.number,
 
-  /** Interaction states, required to calculate change during transform
-   * These should be managed by the map controller and updated via the
-   * `onChangeViewport` callback
-   */
-  startPanLngLat: PropTypes.arrayOf(PropTypes.number),
-  startZoomLngLat: PropTypes.arrayOf(PropTypes.number),
-  startBearing: PropTypes.number,
-  startPitch: PropTypes.number,
-  startZoom: PropTypes.number,
-
   /**
```

Comment:
Now that we have separate markdown docs (which are the source of truth) we can cut and keep the comments here minimal, one liner or better inline comments.